eo-parser/src/main/java/org/eolang/parser/XeListener.java

```diff
@@ -374,9 +387,13 @@ public void enterData(final ProgramParser.DataContext ctx) {
             type = "string";
             data = text.substring(1, text.length() - 1);
         } else if (ctx.TEXT() != null) {
-            type = "string";
+            type = "bytes";
```

Comment:
@andreoss so, a text will produce this XML:

```
<o base="bytes" data="bytes">20-43-4A-5C</o>
```

The information about `string` type is lost in this case. I'm not sure this is the right idea. I think we should keep the attributes untouched. Just change the way we deal with the `text()` content of the XML elements.

Reply:
@yegor256 Fixed `base` type information, now it produces this
```
<o base="string" data="bytes">20-43-4A-5C</o>
```